protocol/scripts/deploys/sepolia/tlc/99-tlc-post-deployment.ts

```diff
@@ -0,0 +1,79 @@
+import { ethers } from 'hardhat';
+import {
+    ensureExpectedEnvvars,
+    mine,
+    setExplicitAccess,
+} from '../../helpers';
+import { connectToContracts, getDeployedContracts } from '../contract-addresses';
+import { ITempleStrategy } from '../../../../typechain';
+
+async function main() {
+    ensureExpectedEnvvars();
+    const [owner] = await ethers.getSigners();
+    const TEMPLE_V2_ADDRESSES = getDeployedContracts();
+    const TEMPLE_V2_INSTANCES = connectToContracts(owner);
+    const EXTERNAL_ALL_USERS = ethers.utils.keccak256(ethers.utils.toUtf8Bytes("EXTERNAL_USER"));
+
+    // Setup Circuit Breaker
+    {
+        // Circuit breaker identifiers
+        await mine(TEMPLE_V2_INSTANCES.CORE.CIRCUIT_BREAKER_PROXY.setIdentifierForCaller(
+            TEMPLE_V2_ADDRESSES.TEMPLE_LINE_OF_CREDIT.ADDRESS,
+            "EXTERNAL_USER"
+        ));
+
+        // Circuit breaker for Temple
+        await mine(TEMPLE_V2_INSTANCES.CORE.CIRCUIT_BREAKER_PROXY.setCircuitBreaker(
+            EXTERNAL_ALL_USERS,
+            TEMPLE_V2_ADDRESSES.CORE.TEMPLE_TOKEN,
+            TEMPLE_V2_ADDRESSES.TEMPLE_LINE_OF_CREDIT.CIRCUIT_BREAKERS.TEMPLE
+        ));
+
+        // Circuit breaker for Dai
+        await mine(TEMPLE_V2_INSTANCES.CORE.CIRCUIT_BREAKER_PROXY.setCircuitBreaker(
+            EXTERNAL_ALL_USERS,
+            TEMPLE_V2_ADDRESSES.EXTERNAL.MAKER_DAO.DAI_TOKEN,
+            TEMPLE_V2_ADDRESSES.TEMPLE_LINE_OF_CREDIT.CIRCUIT_BREAKERS.DAI
+        ));
+
+        // Grant permission for the circuit breaker proxy to call the underlying circuit breakers
+        await setExplicitAccess(
+            TEMPLE_V2_INSTANCES.TEMPLE_LINE_OF_CREDIT.CIRCUIT_BREAKERS.DAI,
+            TEMPLE_V2_ADDRESSES.CORE.CIRCUIT_BREAKER_PROXY,
+            ["preCheck"],
+            true
+        );
+        await setExplicitAccess(
+            TEMPLE_V2_INSTANCES.TEMPLE_LINE_OF_CREDIT.CIRCUIT_BREAKERS.TEMPLE,
+            TEMPLE_V2_ADDRESSES.CORE.CIRCUIT_BREAKER_PROXY,
+            ["preCheck"],
+            true
+        );
+    }
+
+    // Set TLC strategy
+    {   
+        await mine(TEMPLE_V2_INSTANCES.TEMPLE_LINE_OF_CREDIT.INSTANCE.setTlcStrategy(TEMPLE_V2_ADDRESSES.STRATEGIES.TLC_STRATEGY.ADDRESS));
```

Comment:
fyi I found this issue in my PR, `setTlcStrategy` has to be after `addStrategy`

Reply:
Good pickup, thanks. Shows why the deployment scripts on forked mainnet are so useful!

Are you planning on fixing in your branch and will raise a PR to this branch?

Reply:
I'm fixing my branch, I could rebase to this branch, or you could update the pr, I review and the merge to stage, whichever option you prefer :)

Reply:
actually, thinking it better, I'll raise the PR to this branch, will be easier prob